consensus/consensus_v2.go

```diff
@@ -45,7 +45,7 @@ const (
 
 // IsViewChangingMode return true if curernt mode is viewchanging
 func (consensus *Consensus) IsViewChangingMode() bool {
-	return consensus.current.Mode() == ViewChanging
+	return consensus.current.Mode() == ViewChanging || (consensus.current.Mode() == Normal && consensus.IsBackup())
```

Comment:
this is not correct.  

current mode == normal and IsBackup, it can't be in view change mode.  Actually, it shouldn't be in view changing mode at all.  The logic is reversed.